src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -93,14 +93,16 @@ public void parse_compulsoryFieldMissing_failure() {
         // missing phone prefix
         assertParseFailure(parser, NAME_DESC_BOB + VALID_PHONE_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_BOB,
                 expectedMessage);
-
+        /*
```

Comment:
We should delete all the test cases if we no longer need them rather than comment them out.

Reply:
According to my plan in the third suggestion, I will keep this in check in case we need the tests for Optional<> classes. Is that ok or you still think we should delete it now?